app/src/components/fields/MultiAutocompleteFieldVariableSize.tsx

```diff
@@ -109,7 +109,7 @@ const MultiAutocompleteFieldVariableSize: React.FC<IMultiAutocompleteField> = (p
   const { values, touched, errors, setFieldValue } = useFormikContext<IMultiAutocompleteFieldOption>();
 
   const getExistingValue = (existingValues: any[]): IMultiAutocompleteFieldOption[] => {
-    if (!values) {
+    if (!existingValues) {
```

Comment:
question: why was this changed?

Reply:
We ran into a `cant access .includes of undefined` at one point, which put some focus on this component. 

And then, looking at this fucntion now, I don't the previous check was useful, as `values` wasn't used in this function, but exstingValues is.